Разметка всех страниц

Nemchenco96 opened the pull request:
Выполнил разметку главной страницы и каталога

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2065035/tasks/5)

Nemchenco96 committed:
Update index.html

Обновлена разметка главной страницы

Nemchenco96 committed:
Добавление каталога

Добавил каталог с выбором отелей

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,119 @@
+      <section class="main-photo">
+        <h1 class="visually-hidden">Добро пожаловать в Sedona</h1>
+        <img class="main-logo" src="" alt="Основной логотип Sedona.">
+        <h2>СЕДОНА-небольшой городок в Аризоне, заслуживающий большего!</h2>
```
Это вполне может быть заголовком 1-го уровня и тогда придумывать скрытый будет не нужно.

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,119 @@
+      </section>
+
+      <section class="advantages">
+        <h3 class="visually-hidden">5 причин, почему Sedona лучше</h3>
```
Это 2-ой уровень.

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,119 @@
+        <h3 class="visually-hidden">5 причин, почему Sedona лучше</h3>
+        <ul class="advantages-list">
+          <li class="advantages-item">
+            <h4 class="advantages-title">Настоящий городок</h4>
```
А это 3-ий.

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,119 @@
+            <h4 class="advantages-title">Настоящий городок</h4>
+            <p class="advantages-description"> -№1- Седона- не аттракцион для туристов, там течёт своя жизнь</p>
+            <img class="advantages-hotel" src="" alt="Фотография отеля.">
+            <h4>Жильё</h4>
```
Это явно отдельные пункты списка, либо того же самого, либо вложенного.

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,119 @@
+          </li>
+        </ul>
+      </section>
+      <sectionclass="hottel">
```
Перед отправкой на проверку нужно проверь получившийся html с помощью валидатора validator.w3.org/nu/

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,119 @@
+         <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+          <input class="field"  placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email" required>
+            <button class="button newsletter-button" type="submit">
+              <span class="visually-hidden">Подписаться</span>
```
Почему `visually-hidden` ?

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,119 @@
+           <a class="navigation-link" href="">
+             <span class="visually-hidden">Твиттер</span>
+           </a>
+        </li>
```
Явно что-то не так с отступами в разметке.

Dima-Dim commented on catalog.html:
```diff
@@ -0,0 +1,227 @@
+            <ul class="infrastructure-filter-list">
+               <li class="infrastructure-filter-item">
+                  <label class="control">
+                     <input class="control-input" type="checkbox" name="swimming-pool">
```
Тоже проблемы с отступами. Их нужно решить в обоих html файлах.

Dima-Dim commented on catalog.html:
```diff
@@ -0,0 +1,227 @@
+        </fieldset>
+
+          <button class="button" type="submit">Применить</button>
+          <button class="button" type="submit">Сбросить</button>
```
Судя по `type="submit"` не важно что нажмёт пользователь, форма в любом случае отправится :)

Nemchenco96 committed:
Исправление замечаний

1) Заголовки поправил
2) Добавил отдельные пункты списка
3) Через валидатор прогнал, ошибки были только в отсутствии ссылок на картинки.
4) Скрытый класс убрал, пока никакой не стал ставить
5) Отступы поправил
6) Кнопку сброса сделал

Dima-Dim commented on catalog.html:
```diff
@@ -22,15 +24,16 @@
           <li class="navigation-item">
             <a class="navigation-link" href="#">
-              <span class="visually-hidden"> Поиск </span>
+            <span class="visually-hidden"> Поиск </span>
```
Зачем убрал отступ у тега `span` он же вложен в `a`?

Dima-Dim commented on catalog.html:
```diff
@@ -126,100 +130,100 @@ <h3>Найдено гостиниц: 38</h3>
           <ul class="hotels-card">
             <li>
-            <a class="hotels-card-link" href="hotels.html">
+              <a class="hotels-card-link" href="hotels.html">
```
Вот тут нормальные отступы стали.
